src/datepicker/datepicker.component.ts

```diff
@@ -127,4 +128,32 @@ export class DatePicker {
 			}
 		});
 	}
+
+	onInputValueChange(event: string): void {
+		const d = flatpickr.parseDate(event, this.dateFormat, true);
```

Comment:
Hey, we prefer using full words for our variables, maybe something like:

```suggestion
		const eventDates = flatpickr.parseDate(event, this.dateFormat, true);
```